components/turret.py

```diff
@@ -22,6 +25,9 @@ class Turret:
     centre_index: wpilib.DigitalInput
     motor: ctre.WPI_TalonSRX
 
+    MEMORY_CONSTANT: int
+    control_loop_wait_time: float
```

Comment:
This isn't a great name. Waiting is the least important aspect. Perhaps "control_loop_period"?